elastic/kibana
# [Synthetics] handle onboarding for onprem deployments

## Summary

Directs on-prem users to create a private location, before allowing on-prem users to create monitors.

Admin user
[Monitor-Management-Synthetics---Kibana (6).webm](https://user-images.githubusercontent.com/11356435/222554184-3f399764-0c3d-41e4-9652-7ec5616a320c.webm)

User without Fleet privileges
[Synthetics-Getting-Started-Synthetics---Kibana (3).webm](https://user-images.githubusercontent.com/11356435/222554216-893a9a79-a152-459d-b6e6-d5bdfc5014dc.webm)


### Testing

1. Start ES with yarn es snapshot
2. Remove all `xpack.uptime.service` configs in your Kibana.dev.yml
3. Start Kibana connected to local ES
4. Navigate to Synthetics and enable monitor
5. Confirm that Add monitor flow appears first before creating a monitor
